codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/internalconfig/service-2.json

```diff
@@ -0,0 +1,498 @@
+{
+  "version": "2.0",
+  "metadata": {
+    "apiVersion": "2010-05-08",
+    "endpointPrefix": "json-service-endpoint",
+    "globalEndpoint": "json-service.amazonaws.com",
+    "protocol": "rest-json",
+    "serviceAbbreviation": "Json Service",
+    "serviceFullName": "Some Service That Uses Json Protocol",
+    "serviceId": "Json Service",
+    "signingName": "json-service",
+    "signatureVersion": "v4",
+    "uid": "json-service-2010-05-08",
+    "xmlNamespace": "https://json-service.amazonaws.com/doc/2010-05-08/"
+  },
+  "operations": {
```

Comment:
Is there a specific reason the file is relatively long? Could it be shorter? I'm guessing it's a standard json service, but it will probably diverge from the source anyway with time.

Reply:
No, I just copied it from another package. I can remove most operations